src/explore-education-statistics-admin/src/modules/find-statistics/components/EditableDataBlock.tsx

```diff
@@ -16,7 +17,12 @@ const EditableDataBlock = ({ id, onDelete, ...restOfProps }: Props) => {
 
   return (
     <div className={styles.wrapper}>
-      <DataBlock id={id} {...restOfProps} />
+      <DataBlock
```

Comment:
Looks like the releaseId is only being passed to the common DataBlock rendering component so I think we can be confident that the EditableDataBlock will always have access the the ManageReleaseContext. 

Can we grab the releaseId from the context instead of passing it down?